charts/corbusier/templates/deployment.yaml

```
@@ -0,0 +1,110 @@
+      maxSurge: 1
+  selector:
+    matchLabels:
+      {{- include "corbusier.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "corbusier.labels" . | nindent 8 }}
+        app.kubernetes.io/component: backend
+        app.kubernetes.io/part-of: {{ .Chart.Name }}
+      annotations:
+        checksum/config: {{ if .Values.config }}{{ toYaml .Values.config | sha256sum }}{{ end }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)